src/main/java/it/uniroma3/mat/extendedset/intset/ConciseSetUtils.java

```diff
@@ -313,17 +312,17 @@ public static int getLiteralBits(int word)
 
   public static boolean isAllOnesLiteral(int word)
   {
-    return (word & -1) == -1;
+    return word == -1;
   }
 
   public static boolean isAllZerosLiteral(int word)
```

Comment:
Sorry, I pointed at the wrong method, I meant the one below:
```
  public static boolean isAllZerosLiteral(int word)
  {
    return (word | 0x80000000) == 0x80000000;
  }
```
Here for 0 we would have **true**. And now **false**.